### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by fekete-robert**
Replace logo with new Openclarity logo and add a link to the openclarity site

### CODE REVIEW

**Review by martonsereg — CHANGES_REQUESTED** [review]
We don't want the project to be renamed from VMClarity to Vm Security, and this change suggests that. We can still refer to it as VmClarity in the readme here.

We can on the other hand make a 1-sentence mention of OpenClarity and that VmClarity is part of that - and we'll change it later once we get to a unified platform.

**Comment by fekete-robert** [comment]
@martonsereg I've fixed the unneeded renaming

File: README.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ Software Bill Of Materials (SBOM) and security threats such as vulnerabilities,
 
 Join [VMClarity's Slack channel](https://outshift.slack.com/messages/vmclarity) to hear about the latest announcements and upcoming activities. We would love to get your feedback!
 
+VMClarity is now part of the [Openclarity platform](https://openclarity.io).
```
Comment (lgecse) [inline README.md:19]: Please use the `OpenClarity` name instead of `Openclarity`.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="./img/logos/VMClarity-logo-dark-bg-horizontal@4x.png">
-  <source media="(prefers-color-scheme: light)" srcset="./img/logos/VMClarity-logo-light-bg-horizontal@4x.png">
-  <img alt="VMClarity Logo" src="./img/logos/VMClarity-logo-light-bg-horizontal@4x.png">
+  <source media="(prefers-color-scheme: dark)" srcset="https://docs.openclarity.io/img/footer-logos/OC_logo_H_1C_white.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://docs.openclarity.io/img/color-logo/logo.svg">
+  <img alt="Openclarity logo" src="https://docs.openclarity.io/img/color-logo/logo.svg" width="50%">
```
Comment (lgecse) [inline README.md:4]:
```suggestion
  <img alt="OpenClarity logo" src="https://docs.openclarity.io/img/color-logo/logo.svg" width="50%">
```

**Review by lgecse — APPROVED** [review]

**Comment by ramizpolic** [comment]
Outdated, closing this one as we will have to address this separately later on. Thanks!

**Pull request closed without merging**